src/plugins/vis_default_editor/public/components/agg.tsx

```diff
@@ -163,14 +164,16 @@ function DefaultEditorAgg({
 
   const renderAggButtons = () => {
     const actionIcons = [];
+    const aggTitle = agg.type?.title?.toLowerCase();
```

Comment:
what happens if `agg.type?.title?.toLowerCase()` won't exist? Won't we get messages like '{schemaTitle} undefined aggregation has errors'? (not sure)

maybe we could modify this line with:
`const aggTitle = agg.type?.title?.toLowerCase() || ''`

Reply:
Exactly, I have tested it too 🙂